src/pages/pyeongan/Main/components/MainLayout.jsx

```diff
@@ -0,0 +1,137 @@
+import { React, useState } from 'react';
+import '../Main.scss';
+import MainAddComment from './MainAddComment';
+
+export default function MainLayout({ key, item }) {
+  const [commentValue, setCommentValue] = useState('');
+  const [commentArray, setCommentArray] = useState([]);
+
+  const addComment = e => {
+    if (commentValue.length > 0) {
+      const addArray = [...commentArray];
+      addArray.push(commentValue);
+      setCommentArray(addArray);
+      setCommentValue('');
```

Comment:
- 불변성 잘 지키면서 잘 구현해주셨음!
- 기존 배열의 뒤에 새 요소를 추가하고 새로운 배열을 return해주는 concat을 활용하면 다음과 같이 가능
```suggestion
      setCommentArray(commentArray.concat(commentValue));
      setCommentValue('');
```